### DISCUSSION TITLE
请问大家检查更新错误和ocr识别错误是因为什么呢？

### DISCUSSION THREAD

**Discussion by yangnis** [shuax/MouseInc.Settings#339] [General]
别的功能很好了，但就是一点检查更新就提示错误，ocr也没有识别
![image](https://user-images.githubusercontent.com/101557776/187154342-003ed751-0df5-4146-8a2a-c9b6bb628837.png)
![image](https://user-images.githubusercontent.com/101557776/187154366-12daa6b9-a995-4e7e-a59f-cffdfb5e8fb1.png)

**Comment by uiop321** [comment]
更新了，ocr可以用的。

**Reply by andydznj** [reply]
不能使用

**Comment by Eunknight** [comment]
https://github.com/shuax/MouseInc.Settings/discussions/335#discussioncomment-3523123
